docker/Dockerfile.buster

```diff
@@ -0,0 +1,26 @@
+FROM python:3.7-buster AS builder
+
+RUN useradd --user-group --create-home zulip 
+# && apt-get update \
+# && apt-get install -y libffi-dev libxml2-dev libxslt1-dev 
+
+USER zulip
+WORKDIR /home/zulip
+
+RUN set -ex; python3 -m venv zulip-terminal-venv \
+&& . zulip-terminal-venv/bin/activate \
+&& pip3 --disable-pip-version-check install zulip-term \
```

Comment:
We should emphasize that this only installs the version from PyPI, *not* the current version from git master.

Was this your intent? You mention cloning from git, so that is confusing. 

In any case, it would be useful to perhaps have options for building an image from pypi and from git?

Reply:
I used these lines from README. But to actually build in docker you need Dockerfile. Then it depends if you want to build from source or build from PyPI. I've to admit I know very little about Python package system.

Reply:
It would be good to have this work more broadly in future, but for now just indicating it is the latest PyPI version seems important.

I assume the only reason we need the git clone currently is for the `dockerfile` we're using, so that could be worth clarifying too.